### PR TITLE
feat(switch): remove is prefix from props

### LINKED ISSUE #3616: Rename `props`: `is*` eliminates all `is` prefix in the `switch` package
### Description

Change and unify the names of props.
Removing all is prefix from is*

### Problem Statement/Justification

For example, `isOpen` becomes `open` and `isDisabled` becomes `disabled`.

### Proposed Solution or API

_No response_

### Alternatives

props to be changed are marked `@deprecated` in v1

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by kaehehehe**
Closes #3616

## Description

Removing all is prefix from is*.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by nathanlao — COMMENTED** [review]
Thank you for the PR! 

I am wondering if we need to rename the interface for `SwitchIconProps` as well?

https://github.com/yamada-ui/yamada-ui/blob/46fd55826f2a85b52bb277e1a10604ec28e0517c/packages/components/switch/src/switch.tsx#L17-L22


Also, please rename the props in stories if this is necessary:
https://github.com/yamada-ui/yamada-ui/blob/46fd55826f2a85b52bb277e1a10604ec28e0517c/stories/components/forms/switch.stories.tsx#L29-L31

Just a nitpicking! Feel free to fix the name for the test case if you think it's appropriate.
https://github.com/yamada-ui/yamada-ui/blob/46fd55826f2a85b52bb277e1a10604ec28e0517c/packages/components/switch/tests/switch.test.tsx#L56-L57

**Comment by kaehehehe** [comment]
@nathanlao
The Yamada UI team is considering improving the design by eliminating the use of the `is` prefix by default. Therefore, the interface has also removed the `is` prefix.

**Review by kaehehehe — COMMENTED** [review]
@nathanlao 
Thank you for the feedback! I've implemented it. Please check it.

**Review by nathanlao — DISMISSED** [review]
Thank you for letting me know! LGTM!

File: packages/components/switch/src/switch.tsx
Comment (hirotomoyamada) [inline packages/components/switch/src/switch.tsx:1]: Add `active`, `checked`, `focused` and `hovered` to the `props` passed in the `cloneElement`.

Comment (kaehehehe) [reply packages/components/switch/src/switch.tsx:1]: Fixed!

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**